src/System.Net.NetworkInformation/src/System/Net/NetworkInformation/NetworkInformationException.cs

```diff
@@ -0,0 +1,53 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
```

Comment:
I think this file needs to be removed? Doesn't seem to be included in the csproj. (The one in Common, shared between this and the Primitives contract is used instead.)